tasks/JFrogDotnet/task.json

```
@@ -39,17 +39,27 @@
             },
             "options": {
                 "restore": "restore",
-                "push": "NuGet push"
+                "push": "NuGet push",
+                "custom": "custom"
             }
         },
+        {
+            "name": "customCommand",
+            "type": "string",
+            "label": "Custom command",
+            "defaultValue": "",
+            "helpMarkDown": "The custom command which will be passed to .NET Core CLI for execution. Note that commands that require deploying are not supported, and that build-info collection is not available for custom commands.",
+            "required": true,
+            "visibleRule": "command=custom"
+        },
         {
             "name": "arguments",
             "type": "string",
             "label": "Arguments",
             "defaultValue": "",
             "helpMarkDown": "The arguments which will be passed to .NET Core CLI for execution. Arguments should be quoted, and space separated.",
             "required": false,
-            "visibleRule": "command = restore"
+            "visibleRule": "command=restore || command=custom"
```

If you agree, let's do this also is the other fields below
```suggestion
            "visibleRule": "command = restore || command = custom"
```

---

I changed them on purpose, I find it way more readable. Why don't you like this change?